test/extended/util/framework.go

```diff
@@ -275,6 +286,17 @@ func ExtendedTestPath() string {
 	return os.Getenv("EXTENDED_TEST_PATH")
 }
 
+//ArtifactDirPath returns the value of ARTIFACT_DIR environment variable
+func ArtifactDirPath() string {
+	return os.Getenv("ARTIFACT_DIR")
+}
```

Comment:
@gabemontero what happens when this env var is not set? we should at least check this and Fatal if it is empty. Otherwise this will return just empty string, dunno if that is what you want.